content/guides/models/track/runs/grouping.md

```diff
@@ -25,9 +25,57 @@ Pass an optional group and job_type to wandb.init(). This gives you a dedicated
 
 Use `WANDB_RUN_GROUP` to specify a group for your runs as an environment variable. For more on this, check our docs for [**Environment Variables**]({{< relref "/guides/models/track/environment-variables.md" >}})**. Group** should be unique within your project and shared by all runs in the group. You can use `wandb.util.generate_id()` to generate a unique 8 character string to use in all your processes— for example, `os.environ["WANDB_RUN_GROUP"] = "experiment-" + wandb.util.generate_id()`
 
-### 3. Toggle grouping in the UI
+### 3. Set a group in the UI
 
-You can dynamically group by any config column. For example, if you use `wandb.config` to log batch size or learning rate, you can then group by those hyperparameters dynamically in the web app.
+
+After a run is initialized, you can move it to a new group from your workspace or its **Runs** page.
+
+1. Navigate to your W&B project.
+1. Select the **Workspace** or **Runs** tab from the project sidebar.
+1. Search or scroll to the run you want to rename.
+
+    Hover over the run name, click the three vertical dots, then click **Move to another group**.
+1. To create a new group, click **New group**. Type a group name, then submit the form.
+1. Select the run's new group from the list, then click **Move**.
+
+### 4. Toggle grouping by columns in the UI
+
+You can dynamically group by any column, including a column that is hidden. For example, if you use `wandb.config` to log batch size or learning rate, you can then group by those hyperparameters dynamically in the web app. The **Group by** feature is distinct from a [run's run group]({{< relref "grouping.md" >}}). You can group runs by run group. To move a run to a different run group, refer to [Set a group in the UI]({{< relref "#set-a-group-in-the-ui" >}}).
+
+{{% alert title='optional_title' %}}
+In the list of runs, the **Group** column is hidden by default.
+{{% /alert %}}
+
+To group runs by one or more columns:
+
+1. Click **Group**.
+1. Click the names of one or more columns.
+1. If you selected more than one column, drag them to change the grouping order.
+1. Click anywhere outside of the form to dismiss it.
+
+### Customize how runs are displayed
+You can customize how runs are displayed in your workspace from the **Workspace** or **Runs** tab, which share display settings.
```

Comment:
See comment above about this part sounding confusing 

"...which share display settings"